### DISCUSSION TITLE
superplate cli error

### DISCUSSION THREAD

**Discussion by Stevepurpose** [refinedev/refine#4913] [Q&A] [answered]
While following the headless tutorial , I ran the command : npm create refine-app@latest -- -o refine-headless tutorial and  got the message superplate cli has encountered an error   
SAOError: failed to install packages..
Please what can I do?

**Comment by BatuhanW** [comment]
Hello @Stevepurpose which node version are you using? Can you also run the command with `--debug` parameter and send us the output?

`npm create refine-app@latest -- -o refine-headless --debug`

**Comment by Stevepurpose** [comment]
My node version is 20.2.0 .  Ok when I get back to my pc I'd run the
command you specified..

**Comment by Stevepurpose** [comment]
So I ran the code the way you specified and got this screenshot below.

**Reply by w20k** [reply]
There is no screenshot that you've provided :) Could you re-attach it?

**Comment by Stevepurpose** [comment]
I have run the command. Please what do I do next

**Comment by Stevepurpose** [comment]
Sure

**Comment by Stevepurpose** [comment]
sure. Here it is.

**Comment by omeraplak** [comment]
Hey @Stevepurpose ,
Alternatively, you can create a project using playground, https://refine.dev/#playground

**Comment by Stevepurpose** [comment]
Thanks. But  I am confused,how do I go about this? because I just followed
the link and can't see anything pointing me to that.

**Reply by omeraplak** [reply]
Oh okay. Can you share the error you received with `create refine-app`? Please reply to this message not through email, but through GitHub's UI, because we can't see the screenshots you added 👀

**Reply by Stevepurpose** [reply]
Sorry please I need to search for it,will send it as soon as I see the screenshot,it's been a while.

**Reply by Stevepurpose** [reply]
The very first error I encountered was superplate-cli has encountered an error and then I was asked to run it with the debug flag which I did and got this (You can find attached)
![Screenshot (27)](https://github.com/refinedev/refine/assets/105284225/f58feb08-2c8f-43a7-b18c-7850785d46d6)

**Reply by omeraplak** [reply]
Up to this point, no error appears, do you receive an error when you enter your email or leave it blank?

**Reply by Stevepurpose** [reply]
I never entered my email. I thought that it was an alternative way to ask for support

**Reply by Stevepurpose** [reply]
Are you saying I should enter my email?.

**Reply by BatuhanW** [reply, marked as answer]
No, you can just press enter to skip this step. It's not required.
